diff -Nur postgresql-8.2/build-tree/postgresql-8.2beta1/contrib/Makefile postgresql-8.2.new/build-tree/postgresql-8.2beta1/contrib/Makefile
--- postgresql-8.2beta1/contrib/Makefile	2006-09-09 06:07:51.000000000 +0200
+++ postgresql-8.2beta1/contrib/Makefile	2006-09-25 19:32:51.000000000 +0200
@@ -30,7 +30,8 @@
 		spi		\
 		tablefunc	\
 		tsearch2	\
-		vacuumlo
+		vacuumlo	\
+		xml2
 
 ifeq ($(with_openssl),yes)
 WANTED_DIRS += sslinfo
