diff -Nur postgresql-8.3/build-tree/postgresql-8.3beta1/src/backend/main/main.c postgresql-8.3.new/build-tree/postgresql-8.3beta1/src/backend/main/main.c
--- postgresql-8.3beta1/src/backend/main/main.c	2006-09-26 00:50:41.000000000 +0200
+++ postgresql-8.3beta1/src/backend/main/main.c	2006-09-26 00:51:43.000000000 +0200
@@ -326,7 +326,7 @@
 check_root(const char *progname)
 {
 #ifndef WIN32
-	if (geteuid() == 0)
+	if (0 && geteuid() == 0)
 	{
 		write_stderr("\"root\" execution of the PostgreSQL server is not permitted.\n"
 					 "The server must be started under an unprivileged user ID to prevent\n"
diff -Nur postgresql-8.3/build-tree/postgresql-8.3beta1/src/bin/initdb/initdb.c postgresql-8.3.new/build-tree/postgresql-8.3beta1/src/bin/initdb/initdb.c
--- postgresql-8.3beta1/src/bin/initdb/initdb.c	2006-09-26 00:50:41.000000000 +0200
+++ postgresql-8.3beta1/src/bin/initdb/initdb.c	2006-09-26 00:52:09.000000000 +0200
@@ -636,7 +636,7 @@
 
 	pw = getpwuid(geteuid());
 
-	if (geteuid() == 0)			/* 0 is root's uid */
+	if (0 && geteuid() == 0)		/* 0 is root's uid */
 	{
 		fprintf(stderr,
 				_("%s: cannot be run as root\n"
diff -Nur postgresql-8.3/build-tree/postgresql-8.3beta1/src/bin/pg_ctl/pg_ctl.c postgresql-8.3.new/build-tree/postgresql-8.3beta1/src/bin/pg_ctl/pg_ctl.c
--- postgresql-8.3beta1/src/bin/pg_ctl/pg_ctl.c	2006-09-26 00:50:41.000000000 +0200
+++ postgresql-8.3beta1/src/bin/pg_ctl/pg_ctl.c	2006-09-26 00:52:23.000000000 +0200
@@ -1517,7 +1517,7 @@
 	 * Disallow running as root, to forestall any possible security holes.
 	 */
 #ifndef WIN32
-	if (geteuid() == 0)
+	if (0 && geteuid() == 0)
 	{
 		write_stderr(_("%s: cannot be run as root\n"
 					   "Please log in (using, e.g., \"su\") as the "
