diff -ruN postgresql-8.2.3-old/src/include/pg_config_manual.h postgresql-8.2.3/src/include/pg_config_manual.h
--- postgresql-8.2.3-old/src/include/pg_config_manual.h	2004-09-10 16:27:37.000000000 +0200
+++ postgresql-8.2.3/src/include/pg_config_manual.h	2005-06-14 19:21:28.200000000 +0200
@@ -160,7 +160,7 @@
  * here's where to twiddle it.  You can also override this at runtime
  * with the postmaster's -k switch.
  */
-#define DEFAULT_PGSOCKET_DIR  "/tmp"
+#define DEFAULT_PGSOCKET_DIR  "/var/run/postgresql"
 
 /*
  * The random() function is expected to yield values between 0 and
