diff -ruN postgresql-8.2beta4-old/src/Makefile.global.in postgresql-8.2beta4/src/Makefile.global.in
--- postgresql-8.2beta4-old/src/Makefile.global.in	2005-09-27 18:39:32.000000000 +0100
+++ postgresql-8.2beta4/src/Makefile.global.in	2005-10-26 20:40:52.000000000 +0100
@@ -89,7 +89,7 @@
 pkglibdir = $(libdir)
 ifeq "$(findstring pgsql, $(pkglibdir))" ""
 ifeq "$(findstring postgres, $(pkglibdir))" ""
-override pkglibdir := $(pkglibdir)/postgresql
+override pkglibdir := $(pkglibdir)/postgresql/8.3/lib
 endif
 endif
 
