diff -ruN postgresql-8.2.0-old/src/Makefile.global.in postgresql-8.2.0/src/Makefile.global.in
--- postgresql-8.2.0-old/src/Makefile.global.in	2005-11-05 18:54:55.000000000 -0500
+++ postgresql-8.2.0/src/Makefile.global.in	2005-11-05 18:55:59.000000000 -0500
@@ -133,7 +133,7 @@
 
 # These derived path variables aren't separately configurable.
 
-includedir_server = $(pkgincludedir)/server
+includedir_server = $(pkgincludedir)/8.3/server
 includedir_internal = $(pkgincludedir)/internal
 pgxsdir = $(pkglibdir)/pgxs
 
